set cache_valid_time=0 to ensure an apt-get update after the added repo-key

lhoss opened the pull request:
I set cache_valid_time=0 to ensure that an apt-get update after the added repo-key
else you often get 'WARNING: The following packages cannot be authenticated!
See also: http://askubuntu.com/questions/75565/why-am-i-getting-authentication-errors-for-packages-from-an-ubuntu-repository

ps: following might be related ansible issue: https://github.com/ansible/ansible-modules-core/issues/1497
I'm running currently with ansible v1.9.2 (on Mac, installed using brew)

and following role issue might have the same cause:
https://github.com/dj-wasabi/ansible-zabbix-agent/issues/21

dj-wasabi commented:
Thanks!

kostyrevaa commented:
@lhoss why on Earth did you push this very same role to [galaxy](https://galaxy.ansible.com/detail#/role/6905)?

lhoss commented:
Hi!
ok, I removed it again.

My idea was to use ansible galaxy (for the provisioning) with my own fork
as I expect more customizations might be needed.
I'm aware that  for now, my branch is up2date with Werner's (also thx  to
his quick merge of my bug-fix PR), so it looks/ed silly.
So, in case I'm actually extending  my fork, would it be ok/recommended  to
publish on ansible galaxy?

One idea to avoid seeing too many published forks (like I just did) would
be to provide a way to make a role private (non published/searchable).
In any case maybe the Galaxy Doc. should be updated with some 'best
practices' here.

So for now, I will use ansible-galaxy with the  git+https Url dependency.

best,
Laurent

ps: On a side note,  getting following released, will be very very helpful
also:
https://github.com/ansible/ansible/pull/10620
>make ansible-galaxy version work for branches as well as tree-ish



--
Laurent Hoss | TERALYTICS

*Senior Big Data Engineer*

Teralytics AG | Zollstrasse 62 | 8005 Zurich | Switzerland

phone: +41 78 889 83 25
email: laurent.hoss@teralytics.ch
www.teralytics.net

Company registration number: CH-020.3.037.709-7 | Trade register Canton
Zurich

Board of directors: Georg Polzer, Mark Schmitz, Dr. Angelica Kohlmann Küpper
Data Privacy Officer: Prof. Dr. Donald Alan Kossmann
--------------------------------------------
We moved to our new office address: Zollstrasse 62, 8005 Zurich

On Fri, Jan 8, 2016 at 4:03 PM, Александр Костырев <notifications@github.com
> wrote:

> @lhoss <https://github.com/lhoss> why on Earth did you push this very
> same role to galaxy <https://galaxy.ansible.com/detail#/role/6905>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dj-wasabi/ansible-zabbix-agent/pull/22#issuecomment-170025849>
> .
>

kostyrevaa commented:
> So, in case I'm actually extending  my fork, would it be ok/recommended  to
publish on ansible galaxy?

Don't get me wrong!
You ***can*** do whatever you want to do. It's a free world we're living in (ok, not so free but whatever).
I just get upset when I try to search some role in galaxy and I have to scroll over and over again through all those forks of the very same role one guy wrote. And I think to myself maybe that guy abandoned his role but then I get to his github and no, the role is actively supported. And that's very frustrating.
Yeah, there are times when your proposed changes were not merged yet into upstream.
For those cases I have
```
- src: https://github.com/kostyrevaa/ansible-zabbix-agent.git
  name: dj-wasabi.zabbix-agent
  path: roles
```
And then I pursue the upstream's author to get a look at my stuff.
If it gets merged I just change src
```
- src: https://github.com/dj-wasabi/ansible-zabbix-agent.git
```
I just don't see the point.
Even when your proposals are not merged yet why bother to publish it to galaxy.

lhoss commented:
Yeah I agree!
and thx for the tip  (yeah using the 'name' is a good idea)

anyway: I hope  the Ansible ,euhh redhat guys pimp a bit the management
with ansible galaxy..

ps: you have a nice set of role (fork)s on github .. I guess I can 'borrow'
some
like  https://github.com/kostyrevaa/ansible-role-foreman
might be useful soon (I gave it at '*' :)

greets to Russia!
(ps: my wife speaks it, she is from Moldowa .. unfort. I still only
understand <10 words :/ )

--
Laurent Hoss | TERALYTICS

*Senior Big Data Engineer*

Teralytics AG | Zollstrasse 62 | 8005 Zurich | Switzerland

phone: +41 78 889 83 25
email: laurent.hoss@teralytics.ch
www.teralytics.net

Company registration number: CH-020.3.037.709-7 | Trade register Canton
Zurich

Board of directors: Georg Polzer, Mark Schmitz, Dr. Angelica Kohlmann Küpper
Data Privacy Officer: Prof. Dr. Donald Alan Kossmann
--------------------------------------------
We moved to our new office address: Zollstrasse 62, 8005 Zurich

On Fri, Jan 8, 2016 at 5:12 PM, Александр Костырев <notifications@github.com
> wrote:

> So, in case I'm actually extending my fork, would it be ok/recommended to
> publish on ansible galaxy?
>
> Don't get me wrong!
> You *can* do whatever you want to do. It's a free world we're living in
> (ok, not so free but whatever).
> I just get upset when I try to search some role in galaxy and I have to
> scroll over and over again through all those forks of the very same role
> one guy wrote. And I think to myself maybe that guy abandoned his role but
> then I get to his github and no, the role is actively supported. And that's
> very frustrating.
> Yeah, there are times when your proposed changes were not merged yet into
> upstream.
> For those cases I have
>
> - src: https://github.com/kostyrevaa/ansible-zabbix-agent.git
>   name: dj-wasabi.zabbix-agent
>   path: roles
>
> And then I pursue the upstream's author to get a look at my stuff.
> If it gets merged I just change src
>
> - src: https://github.com/dj-wasabi/ansible-zabbix-agent.git
>
> I just don't see the point.
> Even when your proposals are not merged yet why bother to publish it to
> galaxy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dj-wasabi/ansible-zabbix-agent/pull/22#issuecomment-170043428>
> .
>

kostyrevaa commented:
Yeah, galaxy is a... hmm.. you know
